.github/workflows/test.yml

```
@@ -20,17 +20,17 @@ jobs:
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13"]
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v2.2.1
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: make install-test
-    - name: Run tests
-      run: pytest
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)